### DISCUSSION TITLE
想问问一个页里有多个请求的话.send该怎么写？

### DISCUSSION THREAD

**Discussion by Coding-Journey-Studio** [alovajs/alova#238] [Q&A] [answered]
![image](https://github.com/alovajs/alova/assets/15223877/4456bb64-f41b-46e1-8331-dc489eafbaf8)
就如果有多个请求的话，怎么区分？

**Comment by JOU-amjs** [comment, marked as answer]
解构，重命名
![image](https://github.com/alovajs/alova/assets/29848971/7022f30c-cfb5-4964-884c-506837ab641a)
